src/extension/wps/src/main/java/au/org/emii/gogoduck/worker/GoGoDuckModule.java

```diff
@@ -66,75 +142,77 @@ public void postProcess(File file) {
                         subset.get("TIME").start,
                         subset.get("TIME").end)));
 
-        newAttributeList.add(new Attribute("geospatial_lat_min", subset.get("LATITUDE").start));
-        newAttributeList.add(new Attribute("geospatial_lat_max", subset.get("LATITUDE").end));
+        newAttributeList.add(new Attribute(goGoDuckConfig.getLatitudeStart(profile), subset.get("LATITUDE").start));
```

Comment:
The default should be geospatial_lat_min, geospatial_lat_max and so on. if not configured - lets not make them configure the default values